Makefile

```
@@ -85,13 +87,24 @@ go-version-check:
 		[ $(GO_VERSION_MAJ) -eq 1 -a $(GO_VERSION_MIN) -ge 12 ] || (echo "FATAL: Go version should be >= 1.12.x" ; exit 1 ; )
 
 .PHONY: lint
-lint:
+lint: deps
 	# explicitly enable GO111MODULE otherwise go mod will fail
 	GO111MODULE=on go mod tidy && GO111MODULE=on go mod vendor && GO111MODULE=on go mod verify
+	# run go vet
 	$(GO) vet ./...
+	# run go gmt
 	test -z `$(GOFMT) -l $(SKUBA_SRCS)` || { $(GOFMT) -d $(SKUBA_SRCS) && false; }
+	# check terraform fmt
 	$(TERRAFORM) fmt -check=true -write=false -diff=true ci/infra
-	find ci -type f -name "*.sh" | xargs bashate
+	# run golangci-lint
+	$(BINPATH)/golangci-lint run {cmd,internal,pkg}/...
```

Doesn't it work the same if we just:
```suggestion
	$(BINPATH)/golangci-lint run
```

---

Unsure if that would include `vendor`, not sure if `golangci-lint` skips it by default...

---

Yep, since the default config states:
```yaml
# default is true. Enables skipping of directories:
#   vendor$, third_party$, testdata$, examples$, Godeps$, builtin$
skip-dirs-use-default: true
```